search.py

```diff
@@ -28,7 +28,7 @@ def string_to_root(string_in):
     Return: [element]: root element of parsed etree
     """
 
-    return etree.fromstring(string_in)
+    return etree.fromstring(string_in) # pylint: disable=c-extension-no-member
```

Comment:
This is going to pop up repeatedly. The better way to get rid of the error is to disable it in `.pylintrc`. Take a look at [line 53](https://github.com/oxy-compsci/ReChord/blob/master/.pylintrc#L53).

Reply:
Since it's now in `.pylintrc`, the in-line comment doesn't need to be here anymore.